connectors/es/sink.py

```diff
@@ -569,40 +569,29 @@ async def get_access_control_docs(self, generator):
                 self._log_progress()
 
             doc_id = doc["id"] = doc.pop("_id")
-            doc_exists = doc_id in existing_ids
 
-            if doc_exists:
-                last_update_timestamp = existing_ids.pop(doc_id)
-                doc_not_updated = (
-                    TIMESTAMP_FIELD in doc
-                    and last_update_timestamp == doc[TIMESTAMP_FIELD]
-                )
-
-                if doc_not_updated:
-                    continue
-
-                self.total_docs_updated += 1
+            self.total_docs_created += 1
 
-                operation = OP_UPSERT
-            else:
-                self.total_docs_created += 1
-
-                if TIMESTAMP_FIELD not in doc:
-                    doc[TIMESTAMP_FIELD] = iso_utc()
+            if TIMESTAMP_FIELD not in doc:
+                doc[TIMESTAMP_FIELD] = iso_utc()
 
-                operation = OP_INDEX
+            self._logger.debug(f"Creating or updating role with id '{doc_id}'")
+            role_creation_or_update_result = await self.client.client.security.put_role(
+                name=doc_id,
+                cluster=["all"],
+                indices=[
+                    {
+                        "names": ["search-spo"],
```

Comment:
Shouldn't we be able to get the actual index name from `self.index`? This name should be the document index for our full/incremental syncs, so it should be accessible from the connector record's `index_name` if nothing else.